scrapesymbols/gathersymbols.py

```diff
@@ -27,7 +27,6 @@
         '/usr/lib',
     ]
 SYMBOL_SERVER_URL = 'https://s3-us-west-2.amazonaws.com/org.mozilla.crash-stats.symbols-public/v1/'
-MISSING_SYMBOLS_URL = 'https://crash-analysis.mozilla.com/crash_analysis/{date}/{date}-missing-symbols.txt'
```

Comment:
I wonder if we have something similar to this in crash-stats or symbols.mozilla.org, CC @willkg @peterbe

Reply:
Crash Stats doesn't keep track of missing symbols anymore.

Symbols has this API endpoint:

https://symbols.mozilla.org/api/downloads/missing/

with some explanation docs here:

https://tecken.readthedocs.io/en/latest/download.html#s-logged

Is that what you're looking for?

Reply:
As best I can tell, once the gathersymbols.py script had a way to tell that certain symbol files are already present on the symbol server, it no longer needed to check which symbols are missing. So even if I could change the script to interact in a new way with a new server to find missing symbols, I'd say that this was unnecessary. As best I can tell, the whole "missing symbols" business is now entirely redundant.

Or am I misunderstanding what "missing symbols" is all about?

Reply:
The purpose of recording missing symbols was to find out which ones might need to be manually re-fetched from MSDN. In particular, it's for Windows OS system symbols, never for symbols purely related to Firefox. 

It's been a long time since I understood of worked on this but if it's still a mystery and you need more help, I can be convinced to dig through my mental archives a bit more. 

One thing I do remember was the symbols.mozilla.org never ever got a solution to patch the missing symbols from Microsoft. It had the infrastructure in place but never got wind under its wings because there was never a program that could turn certain Windows binaries into symbols. I.e. the `dump_syms` program could never be used because it would never work with 64bit files.

Reply:
Oo--that might get use soon:

https://bugzilla.mozilla.org/show_bug.cgi?id=1588534#c1

calixte is looking into it now.